addons/binding/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -44,7 +44,7 @@
 					"<dc:title>{2}</dc:title>" +
 					"<upnp:class>{3}</upnp:class>" +
 					"<desc id=\"cdudn\" nameSpace=\"urn:schemas-rinconnetworks-com:metadata-1-0/\">" +
-					"RINCON_AssociatedZPUDN</desc>" +
+					"{4}</desc>" +
```

Comment:
Its a good question, look at line 786, by default it still uses that value unless we receive a meta data packet (its been a little while so I'm trying to remember again).  This seems to happen when playing subscription services that have login information associated with them when stored as a favorite.  I run Spotify, Google, Pandora and a I heart Radio stations bookmarked under SONOS  favorites and this works, I have not tried other services.  If we do not get this special meta data packet then it continues to use  RINCON_AssociatedZPUDN.